### ISSUE TITLE
Can't drag minimap-visible-area if you drag out of the window bounds

### ISSUE THREAD

**Issue by mnquintana** [atom-minimap/minimap#320]
If you're dragging the visible area region, and drag out of the bounds of the window, then you "lose" the drag – you have to click and drag on the visible area again to continue dragging. Hopefully this will be more clear with a screencap:

![screencap1](https://cloud.githubusercontent.com/assets/823545/7109036/c8c39e7c-e163-11e4-9c1e-e4aecec3df30.gif)

**Comment by abe33** [comment]
Yes, I explain why in my answer to the comments on this commit: https://github.com/atom-minimap/minimap/commit/d4c02dfb73cda0b852f25720776bb151802407ee 

There's some limitations when dealing with JS mouse events, the choice was between stopping the drag or having unfinished drag gesture when releasing the mouse outside of the window. We're still looking for a better solution that could handle all these cases (there's also the <kbd>alt-tab</kbd> case as described in #314).
